packages/react-core/src/components/Dropdown/DropdownToggle.tsx

```diff
@@ -30,8 +30,10 @@ export interface DropdownToggleProps extends React.HTMLProps<HTMLButtonElement>
   isDisabled?: boolean;
   /** Whether or not the dropdown toggle button should have primary button styling */
   isPrimary?: boolean;
+  /** An image to display within the dropdown toggle */
```

Comment:
I think this description is to broad.  if the consumer provides children, this will be placed before the children.

Reply:
Updated description